client/my-sites/checkout/controller.jsx

```diff
@@ -41,6 +41,7 @@ import UpsellNudge, {
 } from './upsell-nudge';
 import { MARKETING_COUPONS_KEY } from 'calypso/lib/analytics/utils';
 import { TRUENAME_COUPONS } from 'calypso/lib/domains';
+import { hideMasterbar } from 'calypso/state/ui/masterbar-visibility/actions';
```

Comment:
This is a bit of a nitpick, but maybe we could import this as `hideMasterbarAction` or something so that we can reuse `hideMasterbar` without shadowing?

```suggestion
import { hideMasterbar as hideMasterbarAction } from 'calypso/state/ui/masterbar-visibility/actions';
```

Reply:
I agree with Paul, we should bring the same navigation that we have when the user starts creating the store. I think is the main nav, not the checkout one. Regardless these thank you page should have a navigation bar.

Reply:
@JanaMW27 Referring to the [upgrade flow testing video](https://cloudup.com/iFZNuqUvlRd), @ianstewart mentioned skipping over that thank you modal all together and taking the user directly to the setup screen. Would we want to do that instead of reappearing the masterbar?

Reply:
I think having the master bar is OK to have this thank you screen if they don't want to start setting up their store just yet. So with that thank-you screen and the master bar, we allow them to exit this flow and do whatever they can... unless it is actually mandatory to start setting up the store at that moment?

Reply:
I don't believe setting up the store is mandatory at that point. -- The loading screen and the thank you message are technically the same page, so I will need to consider how to implement the reappearing of the masterbar. I'll update once I have it worked out. ☺️

Reply:
As that page is displayed, we could probably call another action to change the state?